backend/app/views/spree/admin/shared/_navigation_footer.html.erb

```diff
@@ -6,9 +6,16 @@
 <% if try_spree_current_user %>
   <ul id="login-nav" class="admin-login-nav">
     <li data-hook="user-account-link">
-      <%= link_to spree.polymorphic_path([:edit, :admin, try_spree_current_user]) do %>
-        <i class='fa fa-user'></i>
-        <%= try_spree_current_user.email %>
+      <% if can?(:admin, try_spree_current_user) %>
+        <%= link_to spree.edit_admin_user_path(try_spree_current_user) do %>
+          <i class='fa fa-user'></i>
+          <%= try_spree_current_user.email %>
+        <% end %>
+      <% else %>
+        <%= link_to spree.admin_path do %>
```

Comment:
I'm not sure it makes sense to link to the root. Maybe we should drop the link entirely (just plain text) if we don't have permissions?

Reply:
fixed!